### PR TITLE
Update 1L, 37, 80 labels

### PR DESCRIPTION
**Pull request by andermatt64**

### CODE REVIEW

File: research/1L.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 ## Description
-
+So far, there are at least 4 variants
 
 ### Carriers observed on
 
 This message label has been observed on the following carriers.
 
-Carrier | IATA Code | ICAO Code
+Carrier | IATA Code | ICAO Code |
```
Comment (kevinelliott) [inline research/1L.md:10]: Is this pipe needed?

**Review by kevinelliott — APPROVED** [review]
Fantastic addition.  Thanks @andermatt64 !

File: research/1L.md (hunk 1 above)
Comment (andermatt64) [reply research/1L.md]: Ohhhh, good catch... originally I added another column but decided against it. Must've forgotten to remove it

Comment (kevinelliott) [reply research/1L.md]: Just proof that I actually look at everything people contribute :)

**Pull request merged**